src/pages/index.js

```diff
@@ -3,3 +3,4 @@ export { HomePage } from "./HomePage/HomePage";
 export { ProductsPage } from "./ProductsPage/ProductsPage";
 export { WishlistPage } from "./WishlistPage/WishlistPage";
 export { PageNotFound } from "./404/404";
+export { SingleProduct } from "./SingleProduct/SingleProduct";
```

Comment:
put an index.js file in the pages folder. import the pages there and from there import into outer pages for better readability.